frappe.go

```diff
@@ -5,7 +5,7 @@ type frappe struct{}
 
 var _ Theme = frappe{}
 
-func (frappe) Name() string { return "frappe" }
+func (frappe) Name() string { return "Frappe" }
```

Comment:
We should probably keep consistency with the Rust equivalent here: https://github.com/catppuccin/rust/blob/main/src/flavour.rs#L28-L33

```suggestion
func (frappe) Name() string { return "frappe" }
```